apps/condo/domains/meter/schema/MeterReadingExportTask.js

```diff
@@ -0,0 +1,257 @@
+const setFileMetaAfterChange = getFileMetaAfterChange(MeterReadingExportTaskFileAdapter, 'file')
+
+
+const MeterReadingExportTask = new GQLListSchema('MeterReadingExportTask', {
+    schemaDoc: 'Stores requested export format, status of export job, link to exported file and information about progress of export job',
+    fields: {
+        status: {
+            schemaDoc: 'Status of the job. Used at client side progress information component',
+            type: 'Select',
+            options: EXPORT_STATUS_VALUES,
+            defaultValue: PROCESSING,
+            isRequired: true,
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: true,
+            },
+        },
+
+        format: {
+            schemaDoc: 'Requested export file format',
+            type: 'Select',
+            options: [EXCEL],
+            isRequired: true,
+            access: {
+                create: true,
+                read: true,
+                update: false,
+            },
+        },
+
+        exportedRecordsCount: {
+            schemaDoc: 'How many records at the moment are exported',
+            type: 'Integer',
+            isRequired: true,
+            defaultValue: 0,
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        totalRecordsCount: {
+            schemaDoc: 'Total records to export that will be determined at server side in export operation',
+            type: 'Integer',
+            isRequired: true,
+            defaultValue: 0,
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        file: {
+            schemaDoc: 'Meta information about file, saved outside of database somewhere. Shape of meta information JSON object is specific to file adapter, used by saving a file.',
+            type: 'File',
+            adapter: MeterReadingExportTaskFileAdapter,
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        meta: {
+            schemaDoc: 'Stores information about query and ids of exported and failed records',
+            type: 'Json',
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        where: {
+            schemaDoc: 'Filtering conditions for records to export',
+            type: 'Json',
+            graphQLInputType: 'MeterReadingWhereInput',
```

Comment:
GraphqlReturnType?

Reply:
`MeterReadingWhereInput` it's input type. We can not use it in `graphqlReturnType`